GDevelop.js/Bindings/postjs.js

```diff
@@ -151,6 +151,7 @@ var adaptNamingConventions = function(gd) {
     //Preserve backward compatibility with some alias for methods:
     gd.VectorString.prototype.get = gd.VectorString.prototype.at;
     gd.VectorPlatformExtension.prototype.get = gd.VectorPlatformExtension.prototype.at;
+    gd.VectorDependencyMetadata.prototype.get = gd.VectorDependencyMetadata.prototype.at;
```

Comment:
Let's remove this :) It was only added for backward compatibility of other classes :)